src/devtools/client/inspector/markup/actions/markup.ts

```diff
@@ -28,9 +31,9 @@ import {
 
 import { paused } from "devtools/client/debugger/src/reducers/pause";
 
-import Highlighter from "highlighter/highlighter";
 import NodeConstants from "devtools/shared/dom-node-constants";
 import { features } from "devtools/client/inspector/prefs";
+import { clearTimeout } from "timers";
```

Comment:
This overrides the browser's `clearTimeout()` function (I guess that VS Code automatically added this import although this is not what we want).

Reply:
Whoops, yup, will fix!